microprofile/opentracing-smallrye/src/main/java/org/wildfly/extension/microprofile/opentracing/smallrye/SubsystemDefinition.java

```diff
@@ -0,0 +1,30 @@
+package org.wildfly.extension.microprofile.opentracing.smallrye;
+
+import java.util.Arrays;
+import java.util.Collection;
+import org.jboss.as.controller.AttributeDefinition;
+import org.jboss.as.controller.PersistentResourceDefinition;
+import org.jboss.as.controller.registry.ManagementResourceRegistration;
+
+public class SubsystemDefinition extends PersistentResourceDefinition {
+    static AttributeDefinition[] ATTRIBUTES = {};
```

Comment:
This should probably be final as I don't see the array being added to anywhere. Or really it could just be deleted.